src/Api/ProductApi.php

```diff
@@ -48,9 +48,11 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
-    public function get(string $code): array
+    public function get(string $code/*, array $queryParameters = [] */): array
     {
-        return $this->resourceClient->getResource(static::PRODUCT_URI, [$code]);
+        $queryParameters = func_num_args() > 1 && is_array(func_get_arg(1)) ? func_get_arg(1) : [];
```

Comment:
Why doing it this way? The method signature can support optional arguments without requiring the interface to have them

Reply:
He seems that you are right, I think I've tested too quickly yesterday.
Will update the PR

Reply:
I was thinking of modifying only the method implementation, but it's even better this way!